README.md

```diff
@@ -2,10 +2,7 @@
 
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-machine-learning.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Build Status](https://github.com/Qiskit/qiskit-machine-learning/workflows/Machine%20Learning%20Unit%20Tests/badge.svg?branch=main)](https://github.com/Qiskit/qiskit-machine-learning/actions?query=workflow%3A"Machine%20Learning%20Unit%20Tests"+branch%3Amain+event%3Apush)[![](https://img.shields.io/github/release/Qiskit/qiskit-machine-learning.svg?style=popout-square)](https://github.com/Qiskit/qiskit-machine-learning/releases)[![](https://img.shields.io/pypi/dm/qiskit-machine-learning.svg?style=popout-square)](https://pypi.org/project/qiskit-machine-learning/)[![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-machine-learning/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-machine-learning?branch=main)<!--- long-description-skip-end -->
 
-The Machine Learning package simply contains sample datasets at present. It has some
-classification algorithms such as QSVM and VQC (Variational Quantum Classifier), where this data
-can be used for experiments, and there is also QGAN (Quantum Generative Adversarial Network)
-algorithm.
+The Machine Learning package contains sample datasets, classification algorithms such as QSVMs (Quantum State Vector Machines), VQCs (Variational Quantum CLassifiers) and QGANs (Quantum Generative Adversarial Networks).
```

Comment:
Probably this intro text ought to be brought more up todate, though I can see this reads a little easier than the old text.